src/olympia/abuse/tests/test_models.py

```diff
@@ -2333,3 +2374,70 @@ def test_notify_reviewer_decision_rejection_addon_already_disabled(self):
             'You may upload a new version which addresses the policy violation(s)'
             not in mail.outbox[0].body
         )
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    'illegal_category,expected',
```

Comment:
I think I'd have just tested based on the constant name?  But you've written it now so 🤷‍♂️